src/main/java/http_server/Router.java

```diff
@@ -29,6 +31,15 @@ public Router() {
         methods.add(Methods.put);
         methods.add(Methods.options);
         methods.add(Methods.delete);
+        repository = new FileRepository();
```

Comment:
I wanted to set a default File handler as I thought it would be the main way for handling a repository in the application. For the edge cases where we are testing, the repository can be overwritten using the `setRepository` method